runtime/src/lib.rs

```diff
@@ -1405,47 +1587,48 @@ construct_runtime!(
-        Session: pallet_session::{Module, Call, Storage, Event, Config<T>},
-        PoAModule: poa::{Module, Call, Storage, Config<T>},
-        Grandpa: grandpa::{Module, Call, Storage, Config, Event},
-        Authorship: pallet_authorship::{Module, Call, Storage},
-        TransactionPayment: transaction_payment::{Module, Storage},
-        Utility: pallet_utility::{Module, Call, Event},
-        BbsPlus: bbs_plus::{Module, Call, Storage, Event},
-        DIDModule: did::{Module, Call, Storage, Event, Config},
-        Revoke: revoke::{Module, Call, Storage, Event},
-        BlobStore: blob::{Module, Call, Storage},
-        Master: master::{Module, Call, Storage, Event<T>, Config},
-        Sudo: sudo::{Module, Call, Storage, Event<T>, Config<T>},
-        MigrationModule: token_migration::{Module, Call, Storage, Event<T>},
-        Anchor: anchor::{Module, Call, Storage, Event<T>},
-        Attest: attest::{Module, Call, Storage},
-        Democracy: pallet_democracy::{Module, Call, Storage, Event<T>},
-        Council: pallet_collective::<Instance1>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>},
-        TechnicalCommittee: pallet_collective::<Instance2>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>},
-        TechnicalCommitteeMembership: pallet_membership::<Instance1>::{Module, Call, Storage, Event<T>, Config<T>},
-        Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
-        Ethereum: pallet_ethereum::{Module, Call, Storage, Event, Config, ValidateUnsigned},
-        EVM: pallet_evm::{Module, Config, Call, Storage, Event<T>},
-        PriceFeedModule: price_feed::{Module, Call, Storage, Event},
-        FiatFilterModule: fiat_filter::{Module, Call},
-        AuthorityDiscovery: pallet_authority_discovery::{Module, Call, Config},
-        Historical: pallet_session_historical::{Module},
-        ImOnline: pallet_im_online::{Module, Call, Storage, Event<T>, ValidateUnsigned, Config<T>},
-        Babe: pallet_babe::{Module, Call, Storage, Config, ValidateUnsigned},
-        Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>, ValidateUnsigned},
-        ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Module, Call, Storage, Event<T>, ValidateUnsigned},
-        Offences: pallet_offences::{Module, Call, Storage, Event},
-        Treasury: pallet_treasury::{Module, Call, Storage, Config, Event<T>},
-        Bounties: pallet_bounties::{Module, Call, Storage, Event<T>},
-        StakingRewards: staking_rewards::{Module, Call, Storage, Event<T>},
-        Elections: pallet_elections_phragmen::{Module, Call, Storage, Event<T>, Config<T>},
-        Tips: pallet_tips::{Module, Call, Storage, Event<T>},
-        Identity: pallet_identity::{Module, Call, Storage, Event<T>},
-        Accumulator: accumulator::{Module, Call, Storage, Event},
+        System: system::{Pallet, Call, Config, Storage, Event<T>},
```

Comment:
Can you add indices for modules like https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L1408? This will help in minimizing future changes in Ledger wallet app